app/web/features/communities/newEvents/EventItem.tsx

```diff
@@ -0,0 +1,146 @@
+import {
+  Card,
+  CardContent,
+  CardMedia,
+  Theme,
+  Tooltip,
+  Typography,
+} from "@material-ui/core";
+import { eventImagePlaceholderUrl } from "appConstants";
+import Pill from "components/Pill";
+import dayjs from "dayjs";
+import { useAuthContext } from "features/auth/AuthProvider";
+import { useTranslation } from "i18n";
+import { COMMUNITIES, GLOBAL } from "i18n/namespaces";
+import Link from "next/link";
+import { Event } from "proto/events_pb";
+import { routeToEvent } from "routes";
+import { timestamp2Date } from "utils/date";
+import makeStyles from "utils/makeStyles";
+
+const useStyles = makeStyles((theme: Theme) => ({
+  attendees: {
+    display: "flex",
+    alignItems: "flex-end",
+  },
+  card: {
+    display: "flex",
+    width: "100%",
+    height: theme.spacing(20),
+    margin: theme.spacing(2, 0, 2, 0),
+    border: `1px solid ${theme.palette.grey[300]}`,
+    borderRadius: theme.spacing(1),
+  },
+  cardMedia: {
+    height: "100%",
+    width: "25%",
+  },
+  cardContent: {
+    width: "75%",
+    padding: theme.spacing(2, 4),
+    display: "flex",
+    flexDirection: "column",
+    justifyContent: "space-between",
+  },
+  eventInfo: {
+    display: "flex",
+    justifyContent: "flex-end",
+    flexDirection: "column",
+  },
+  row: {
+    display: "flex",
+    justifyContent: "space-between",
+  },
+  tags: {
+    gap: theme.spacing(1),
+  },
+  title: {
+    display: "-webkit-box",
+    boxOrient: "vertical",
+    lineClamp: 2,
+    overflow: "hidden",
+  },
+}));
+
+const EventItem = ({ event }: { event: Event.AsObject }) => {
+  const classes = useStyles({
+    eventImageSrc: event.photoUrl || eventImagePlaceholderUrl,
+  });
+  const { t } = useTranslation([GLOBAL, COMMUNITIES]);
+
+  const startTime = dayjs(timestamp2Date(event.startTime!)).format("llll");
```

Comment:
what about:
- not showing the year when the event is upcoming
- show it when passed already?

not a request, but what do you think about it?

Reply:
another thing (this could be a petition xd)

what about this formatting?
![image](https://github.com/user-attachments/assets/fc56ee9e-f510-46f6-93f6-645f5ae35969)

day_of_week, day, month **at** hour

Reply:
So I like this, but we're using the `LocalizedFormat` plugin to localize the dates for dayjs and it doesn't support that:

https://day.js.org/docs/en/display/format#list-of-localized-formats

I think it's not worth all the work it would take to get it to work, but I do agree I like that more (I'd also love to use the Calendar plugin to do `Today`, `Tomorrow`, `Yesterday` etc) but no localization support.

Reply:
Oh also, I decided to just show the `startTime` in the `EventsList` I think it's good just to have the most important data in the short version, then they can see the extended data in the event. What do you think?

Reply:
fine with that :)